src/components/Editor/index.js

```diff
@@ -234,6 +234,13 @@ class Editor extends PureComponent<Props, State> {
     // responsible for updating the editor annotations.
     const { selectedLocation, selectedSource } = this.props;
 
+    // To stop newly opened sources from inheriting the scroll-state
+    // of the currently open tab we force a scrollTo
+    if (selectedSource && selectedSource.get("loadedState") != "loaded") {
+      const newSource = document.querySelector(".CodeMirror-scroll");
+      /*::`*/ if (newSource) newSource.scrollTo(0, 0); /*::`*/
```

Comment:
our linter is complaining about these inline comments, im also wondering what do you want to express with them? maybe there is a better way?

im also wondering if there are any edge cases for this. We don't always want to open a source and scroll to 0,0. sometimes we want to be at a line. maybe what we want to do, when we select location, is default to 0 if no other location is specified?

Reply:
The linter is complaining because it didn't expect the property .toTop on an object that might not be there. Looking for a solution to this I've moved the code to a better place and rewritten it,  working on disabling the scroll if there is a location set now.

Reply:
hmm this is the error on the ci (lint usually doesnt complain about properties on objects, do you mean flow?):

```
yarn lint-js

yarn lint-js v0.28.4
$ eslint *.js "src/**/*.js" --fix
The babel/generator-star-spacing rule is deprecated. Please use the built in generator-star-spacing rule instead.

/home/ubuntu/debugger.html/src/components/Editor/index.js
  241:7  error  Unexpected comment inline with code  no-inline-comments
  243:9  error  Unexpected comment inline with code  no-inline-comments

✖ 2 problems (2 errors, 0 warnings)

error Command failed with exit code 1.

yarn lint-js returned exit code 1
```

Reply:
an str for opening a new file at a location is to have breakpoints in a few files, close all of them, and click the breakpoint

Reply:
Sounds like a great str, thanks!